### ISSUE TITLE
Install file is damaged

### ISSUE THREAD

**Issue by molonlabe88** [yermak/AudioBookConverter#528]
I receive this error “AudiobookConverter” is damaged and can’t be opened. You should eject the disk image. 

Running 14.1.2 Apple M1 Pro chip.

**Comment by cerinawithasea** [comment]
you might have to try this
Open Terminal enter commands

cd /Applications/AudiobookConverter.app

./Contents/MacOS/AudiobookConverter

**Comment by krejko** [comment]
I tried using the terminal commands above and received the same file is damaged error.

**Comment by AdamMescher** [comment]
I'm on an Intel 2016 15-inch MBP and got the same error after downloading the .dmg and moving AudiobookConverter to the `Applications` directory.

I deleted the application, re-opened the downloaded .dmg file, moved AudiobookConverter to the Applications directory using the ui, and then ran the two terminal commands listed within the [installation instructions listed in the wiki](https://github.com/yermak/AudioBookConverter/wiki/Installation) and I was able to successfully launch the application. 

Kudos to @molonlabe88 for asking the question and @cerinawithasea for answering. It allowed me to keep going on what I wanted to get done.

**Comment by welldunndesign** [comment]
After following these steps I'm met with this:

Last login: Fri Jan 19 11:43:02 on ttys001
joeldunn@Joels-MacBook-Pro ~ % cd /Applications/AudiobookConverter.app

./Contents/MacOS/AudiobookConverter
zsh: killed     ./Contents/MacOS/AudiobookConverter
joeldunn@Joels-MacBook-Pro AudiobookConverter.app % 


Any advice? I know nothing about using terminal so have zero clue how to troubleshoot this.

**Comment by nathenxbrewer** [comment]
Same here. I followed the instructions to launch from the console but still get the same error saying it's damaged. 
![image](https://github.com/yermak/AudioBookConverter/assets/43285863/e952e33e-64e8-46fb-ab68-9e9d12a5d66d)

**Comment by QuantumForced** [comment]
yep same here like in nathenxbrewer's screenshot, I have a MBP M1 running MacOS Sonoma 14.1.2 (23B92)

**Comment by QuantumForced** [comment]
I have found a fix! For those still getting the 'damaged' error despite running the terminal commands in the wiki, here's how to get it to launch successfully:

1. Download the latest release .dmg file
2. Open terminal and paste in `xattr -cr ` (note the required whitespace after -cr)
3. Drag the downloaded .dmg file into the same terminal window & hit enter (nothing will happen, thats normal)
4. Mount (double click) the same downloaded .dmg file (you should then see the app + the Applications folder)
5. Repeat steps 2+3, but instead of dragging in the .dmg, drag in the now mounted AudiobookConverter.app file, hit enter)
6. Drag the AudiobookConverter.app file into the Applications folder, open the Applications folder and right click > Open the AudiobookConverter app - done! (I didn't need to run the commands in the wiki at this stage)

This is what my two commands looked like for reference: 
`xattr -cr /Users/quantum/Downloads/AudiobookConverter-6.2.10.dmg`
`xattr -cr /Volumes/AudiobookConverter/AudiobookConverter.app`

Typed these instructions out in layman terms in case it helps anyone

Enjoy

**Comment by nathenxbrewer** [comment]
> I have found a fix! For those still getting the 'damaged' error despite running the terminal commands in the wiki, here's how to get it to launch successfully:
> 
> 1. Download the latest release .dmg file
> 2. Open terminal and paste in `xattr -cr ` (note the required whitespace after -cr)
> 3. Drag the downloaded .dmg file into the same terminal window & hit enter (nothing will happen, thats normal)
> 4. Mount (double click) the same downloaded .dmg file (you should then see the app + the Applications folder)
> 5. Repeat steps 2+3, but instead of dragging in the .dmg, drag in the now mounted AudiobookConverter.app file, hit enter)
> 6. Drag the same mounted AudiobookConverter.app file into the Applications folder, open the Applications folder and **right click > Open** the AudiobookConverter app - done! (I didn't need to run the commands in the wiki at this stage)
> 
> This is what my two commands looked like for reference: `xattr -cr /Users/quantum/Downloads/AudiobookConverter-6.2.10.dmg` `xattr -cr /Volumes/AudiobookConverter/AudiobookConverter.app`
> 
> Typed these instructions out in layman terms in case it helps anyone
> 
> Enjoy

This aligns with what I've found when recently installing some macOS apps from....other sources. They include a script titled "Open Gatekeeper friendly" that executes these commands for you and allows you to drag the app into your application folder, bypassing Gatekeeper. It seems macOS likes to pretend that the install file is 'damaged' when in reality it's blocking the install. 

<img width="511" alt="image" src="https://github.com/yermak/AudioBookConverter/assets/43285863/1ba277ac-ef4c-41e8-b4cc-b437bf5cf8d3">

**Comment by QuantumForced** [comment]
> > I have found a fix! For those still getting the 'damaged' error despite running the terminal commands in the wiki, here's how to get it to launch successfully:
> > 
> > 1. Download the latest release .dmg file
> > 2. Open terminal and paste in `xattr -cr ` (note the required whitespace after -cr)
> > 3. Drag the downloaded .dmg file into the same terminal window & hit enter (nothing will happen, thats normal)
> > 4. Mount (double click) the same downloaded .dmg file (you should then see the app + the Applications folder)
> > 5. Repeat steps 2+3, but instead of dragging in the .dmg, drag in the now mounted AudiobookConverter.app file, hit enter)
> > 6. Drag the same mounted AudiobookConverter.app file into the Applications folder, open the Applications folder and **right click > Open** the AudiobookConverter app - done! (I didn't need to run the commands in the wiki at this stage)
> > 
> > This is what my two commands looked like for reference: `xattr -cr /Users/quantum/Downloads/AudiobookConverter-6.2.10.dmg` `xattr -cr /Volumes/AudiobookConverter/AudiobookConverter.app`
> > Typed these instructions out in layman terms in case it helps anyone
> > Enjoy
> 
> This aligns with what I've found when recently installing some macOS apps from....other sources. They include a script titled "Open Gatekeeper friendly" that executes these commands for you and allows you to drag the app into your application folder, bypassing Gatekeeper. It seems macOS likes to pretend that the install file is 'damaged' when in reality it's blocking the install.
> 
> <img alt="image" width="511" src="https://private-user-images.githubusercontent.com/43285863/301206636-1ba277ac-ef4c-41e8-b4cc-b437bf5cf8d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY3MTM2MTksIm5iZiI6MTcwNjcxMzMxOSwicGF0aCI6Ii80MzI4NTg2My8zMDEyMDY2MzYtMWJhMjc3YWMtZWY0Yy00MWU4LWI0Y2MtYjQzN2JmNWNmOGQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTMxVDE1MDE1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkNDk4NmYxOTM4MGNjZmIwZGM0MmU3M2ZlY2E2ZjU4OTA5ODQ3NDBlZmM2ZTA0NDM3MTM3YTUzZGZiYmYyOTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wKiYcgj33zMUQ_tUOLvpdXZGVRjiLp1PpIi3oodL_58">

It looks like the command used by the group in question is `sudo xattr -rd com.apple.quarantine drag_here_the.app`, I just tried this command instead of the one I previously suggested (although with this one I had to right click > open twice before it would allow me to actually open) and they both appear to get AudioBookConverter to open, however, they both fail to add a file/folder via dragging or using the app menu.. nothing actually gets added. Is this the case for anyone else, or just me? 

Launching the app via terminal following the instructions in the wiki (after running the xattr commands), shows Java exceptions exactly when trying to add a file/folder. I wonder if this issue is related to having an apple silicon chip (Mx)?

**Comment by ddauerbach** [comment]
Sorry to say that none of these fixes worked for me.  Tried them twice. Later, when I have time I'll give it another go. But this is sad.

**Comment by OptimusGREEN** [comment]
Try holding ctrl then open from the context menu, the box gives an option to open anyway.

**Comment by 4nur4g** [comment]
For me xattr method didn't work. Instead, after installation, I tried to open the file. Instead of moving to the bin, I closed it. Then, I went to my Mac's settings. There, I went to Privacy & security tab. There, I enabled "open anyway". It worked!

**Comment by novikor** [comment]
Have a fix!
<img width="372" alt="image" src="https://github.com/yermak/AudioBookConverter/assets/26086561/9e9861bc-f39f-4e74-be3a-df8872f929e5">
It was as simple as just reading the MacOS help window :D 
![image](https://github.com/yermak/AudioBookConverter/assets/26086561/8ed72043-0044-46ee-89d5-ad9996c7947e)

Just used Control + Click on the App, then "Open". And that's all

**Comment by novikor** [comment]
> Try holding ctrl then open from the context menu, the box gives an option to open anyway.

+, that works
